### PR TITLE
CLOUDP-121631: Changing order to get any closed alert to reduce flakiness

### PR DESCRIPTION
**Pull request by colm-quinn**
## Proposed changes

Current E2E relies on an open alert being present on the env, which cannot be guaranteed. Changing the tests to instead rely on a closed alert, of which at least 1 should always be present.

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by blva — APPROVED** [review]
LGTM

File: e2e/atlas/alerts_test.go [hunk 1]
```diff
@@ -60,10 +60,12 @@ func TestAlerts(t *testing.T) {
 		}
 	})
 
-	t.Run("List with no status", func(t *testing.T) {
+	t.Run("List with status OPEN", func(t *testing.T) {
 		cmd := exec.Command(cliPath,
 			alertsEntity,
 			"list",
+			"--status",
+			"OPEN",
```
Comment (blva) [inline e2e/atlas/alerts_test.go:68]: nit:  we have an `open` const

Comment (colm-quinn) [reply e2e/atlas/alerts_test.go:68]: The const is also wrong :) Refactoring

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,21 +43,23 @@ we have a set of guidelines to ensure that all contributions are acceptable.
...
+  - Run `make fmt`.
+  - Add documentation if you are adding new features or changing functionality.
+  - Confirm that `make check` succeeds. [GitHub Actions](https://github.com/mongodb/mongocli/actions).
 
 ### Development Setup
 
 #### Prerequisite Tools
+
 - [Git](https://git-scm.com/)
-- [Go (at least Go 1.17)](https://golang.org/dl/)
```
Comment (colm-quinn) [inline CONTRIBUTING.md:58]: [flyby] Updating to match go.mod, the rest of the doc is auto formatted by my IDE

**Review by gssbzn — APPROVED** [review]

**Pull request merged**